docs/modules/servers/pages/index.adoc

```diff
@@ -82,3 +83,8 @@ This server is:
  * The most feature-rich server, but also by far the most complex
 
 
+
+[#test]
+== James Test Server
+
+The xref:test.adoc[*Test Server*] is, as the name suggests, for testing purposes.
```

Comment:
This server holds all of its data in memory, is easy and quick to boot.

Reply:
Thanks! I added this sentence.